bitnami/redis/values-production.yaml

```diff
@@ -712,3 +712,11 @@ podSecurityPolicy:
   ## Specifies whether a PodSecurityPolicy should be created
   ##
   create: false
+
+## Define a disruption budget
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+##
+podDisruptionBudget:
+  enabled: true
```

Comment:
Thi

> If the feature is disabled by default, it shouldn't be a problem with the default configuration (1 master, 2 slaves), right?

This change doesn't create problems with default configuration. It just seems that behaviour is not changed at all

Reply:
Well, depending on the specific use case, users will be able to set desired values for PDB configuration. This PR eases that.

It is true that the original issue is not going to be solved, and still slaves and master will be shutting down at the same time. But it can be addressed in a different Issue/PR. To be honest, I can't think of a straightforward solution for that. Suggestions are welcome!